src/options/options_handler.cpp

```diff
@@ -1203,6 +1203,48 @@ theory::bv::SatSolverMode OptionsHandler::stringToSatSolver(std::string option,
   }
 }
 
+const std::string OptionsHandler::s_bvProofFormatHelp =
+    "\
+Proof formats currently supported by the --bv-proof-format option:\n\
+\n\
+  lrat : DRAT with unit propegation hints to accelerate checking (default)\n\
+\n\
+  drat : Deletation and Resolution Asymetric Tautology Additions \n\
```

Comment:
```suggestion
  drat : Deletion and Resolution Asymetric Tautology Additions \n\
```

Otherwise the descriptions look good to me :)

Reply:
Haha. Good spot!